core/tauri-utils/src/config.rs

```diff
@@ -1223,6 +1223,26 @@ impl Default for DisabledCspModificationKind {
   }
 }
 
+fn de_remote_domain_access_scope_windows<'de, D>(deserializer: D) -> Result<Vec<String>, D::Error>
+where
+  D: Deserializer<'de>,
+{
+  let windows = Option::<Vec<String>>::deserialize(deserializer)?.unwrap_or_default();
+  for w in &windows {
+    if w == "*" || w == "**" {
```

Comment:
I think this check is not sufficient as devs can just write `?*` or `[?*]` to bypass this full wildcard prevention check.
Also from my perspective allowing `**` anywhere doesn't make sense.  The `glob` crate is imho meant for  file paths and we should use a more simpler approach here.

From my perspective we should use simple string comparisons like `starts_with ` or `ends_with ` to allow something like `foo-` or `-remote` for labels instead of using complex matching pattern. We could expose these two patterns via a flag (e.g: `"fuzzy_matching" : true`)

If there is a reason to go for regex patterns I'm all ears but my assumption is most devs want to allow remote access to certain window types where they can prefix with `remote-`. Using the glob crate would allow for more fine grained matching but will be a footgun for most devs.